host-bmc/host_pdr_handler.cpp

```diff
@@ -1856,6 +1856,8 @@ void HostPDRHandler::createDbusObjects()
                 CustomDBus::getCustomDBus().setSlotType(
                     entity.first,
                     "xyz.openbmc_project.Inventory.Item.PCIeSlot.SlotTypes.OEM");
+                CustomDBus::getCustomDBus().setlinkreset(
```

Comment:
setLinkReset - camel case usage?

Reply:
This needs changes at multiple other places, can we take this as separate commit?

Reply:
Done